### DISCUSSION TITLE
Stream: Deploying from a free Azure account

### DISCUSSION THREAD

**Discussion by pamelafox** [microsoft/AI-Chat-App-Hack#27] [Announcements]
As promised, I put together a video of deploying from a free Azure account:

https://www.youtube.com/watch?v=nlIyos0RXHw

I did end up having to redo part of my provision due to leftover resources from an earlier deploy, but that might be helpful for anyone else that needs to redo for related reasons.

**Comment by pallavigadepalli** [comment]
thanks Pamela just starting to use this for low or no code deployment. thank you! I will reach out if I have any questions.

**Reply by pamelafox** [reply]
That video is deploying https://github.com/Azure-Samples/azure-search-openai-demo/, so I'd consider that fairly high code. We try to make deployment as easy as possible, but it's still fairly complex. 
If you're having issues deploying it, you could start with the Azure AI Studio On Your Data, hooking that up to a Blob Storage, for example.
